packages/codec/lib/compilations/utils.ts

```diff
@@ -11,7 +11,7 @@ import * as Common from "@truffle/compile-common";
 import { Compilation, Contract, Source, VyperSourceMap } from "./types";
 
 export function shimCompilations(
-  inputCompilations: Common.Compilation[],
+  inputCompilations: (Common.Compilation & { contracts: Common.EvmCompiledContract[] })[],
```

Comment:
Could we assign some type name to this so we're not writing this out longhand all the time?

Reply:
This was the easier way I came across to represent it given that it only needs to be specified twice.

Otherwise it involves creating an EvmCompilation type that contains EvmCompiledContract. It's doable, but I'd argue it's more cumbersome for these two instances.

Reply:
In retrospective. I've extracted the type at the top of the file without exporting it. That way it can be reused without repeating it at all.